src/codemagic/models/certificate_p12_exporter.py

```diff
@@ -54,6 +54,9 @@ def _is_noenc_flag_supported(self) -> bool:
                 capture_output=True,
                 check=False,
             )
+            # Check both stdout and stderr because LibreSSL doesn't have help commands per-se.
+            # Execution fails, and it just outputs "unknown option '-help'" along with full
```

Comment:
for the sake of accuracy, the output is `unknown option 'help'` (no dash)

Reply:
Here the output is indeed `"unknown option '-help'"`, because the `openssl pkcs12` is called with `-help` argument that OpenSSL supports.

Reply:
true